rpa-home/modules/ROOT/pages/ms-automation-credits-2.adoc

```diff
@@ -0,0 +1,21 @@
+= MuleSoft Automation Credits 2.0
```

Comment:
I think we should try to focus on the JTBD instead of the feature. 
Said that I can not think of better titles than "Learning about...", "Getting to know...", etc.